aspnet/mvc/controllers/testing/sample/TestingControllersSample/src/TestingControllersSample/Controllers/SessionController.cs

```diff
@@ -0,0 +1,36 @@
+﻿using Microsoft.AspNet.Mvc;
+using TestingControllersSample.Core.Interfaces;
+using TestingControllersSample.ViewModels;
+
+namespace TestingControllersSample.Controllers
+{
+    public class SessionController : Controller
+    {
+        private readonly IBrainStormSessionRepository _sessionRepository;
+
+        public SessionController(IBrainStormSessionRepository sessionRepository)
+        {
+            _sessionRepository = sessionRepository;
+        }
+
+        public IActionResult Index(int? id)
+        {
+            if (!id.HasValue)
+            {
+                return RedirectToAction("Index","Home",null);
```

Comment:
Should fix spacing here, plus recommend using named param for the `null` value. I'm a little surprised that parameter is needed at all - can it just be omitted?